site/gatsby-site/src/api/parseNews.js

```diff
@@ -21,3 +22,51 @@ export default async function handler(req, res) {
 
   res.status(200).json(response);
 }
+
+const getArticle = async (url, config) => {
+  try {
+    const parserConfig = { contentType: 'markdown' };
+
+    if (config.cookies) {
+      parserConfig.html = await getHtmlWithCookies(url);
+    }
+
+    const article = await Parser.parse(url, parserConfig);
+
+    return article;
+  } catch (error) {
+    if (config.cookies) {
+      throw error;
```

Comment:
Since this error is facing editors, is this the best way to surface the error? would a toast be more helpful to a non-dev user? And would rollbar catch this?